src/core/core.js

```diff
@@ -133,6 +133,7 @@ export default class Core {
       experienceKey: this._experienceKey,
       locale: this._locale,
       experienceVersion: this._experienceVersion,
+      visitor: this.storage.get(StorageKeys.VISITOR),
```

Comment:
like Yen said it's probably unnecessary to store the visitor in storage, and we already have a few too many storage keys